gst/nnstreamer/tensor_sink/tensor_sink.c

```diff
@@ -282,6 +282,12 @@ gst_tensor_sink_init (GstTensorSink * self)
 
   /** enable qos */
   gst_base_sink_set_qos_enabled (bsink, DEFAULT_QOS);
+
+  /**
+   * Default FALSE (synchronize on the clock)
+   * It may be delayed with tensor_filter element, to invoke neural network model.
+   */
+  gst_base_sink_set_sync (bsink, FALSE);
```

Comment:
How about declaring `DEFAULT_SYNC` to maintain the consistency with other codes?

Reply:
updated. please see the changes.

Reply:
LGTM
When I tested with this PR, the problem was resolved clearly!